packages/newspringchurchapp/src/prayer/PrayerList/index.js

```diff
@@ -129,11 +129,12 @@ class PrayerList extends PureComponent {
                 >
                   {(flagPrayer) => (
                     <PrayerCard
-                      avatarSize={'medium'}
                       onPress={this.handleOnPress}
+                      expanded
+                      avatarSize={'medium'}
                       options={[
                         {
-                          title: 'Flag as Inappropriate',
+                          title: 'Flag as inappropriate',
```

Comment:
My only pushback here (and why I changed it in a different PR) is that I feel like menu items should be titles not sentences. But that's beyond the scope of getting things done today.